spec/requests/tool_integration/baby_cost_calculator_spec.rb

```diff
@@ -1,14 +1,29 @@
-RSpec.describe ToolMountPoint::BabyCostCalculator, type: :request do 
+RSpec.describe ToolMountPoint::BabyCostCalculator, type: :request do
   %W(
-    /en/tools/#{ToolMountPoint::BabyCostCalculator::EN_ID}
-    /cy/tools/#{ToolMountPoint::BabyCostCalculator::CY_ID}
-  ).each do |path|
-    describe path do
-      before do
-        get path
+    en
+    cy
+  ).each do |locale|
+    context "locale #{locale}" do
+      let (:path) { "/#{locale}/tools/" + "ToolMountPoint::BabyCostCalculator::#{locale.upcase}_ID".constantize }
+
+      describe 'non-syndicated tool request' do
+        before do
+          get path, nil, { 'X-Syndicated-Tool' => false }
```

Comment:
Do we still get the header at all on non syndicated requests? If not, might be appropriate to make the test mimic that.